doc/source/dev/gitwash/index.rst

```diff
@@ -10,6 +25,9 @@ Contents:
 
    git_intro
    following_latest
-   git_development
-   development_workflow
+   development_setup
+   configure_git
+   dot2_dot3
```

Comment:
What is this? I cannot find it in the current doc tree.  If related to python2, it should be removed.

Reply:
It is related to difference in git between `ref1..ref2` to `ref1...ref2`

http://www.numpy.org/devdocs/dev/gitwash/dot2_dot3.html

Reply:
OK, thanks, learn something new every day!